tests/ui-api-layer-acceptance-tests/domain/remoteenvironment/remoteenvironment_test.go

```diff
@@ -3,83 +3,226 @@
 package remoteenvironment
 
 import (
+	"fmt"
 	"testing"
+	"time"
+
+	"github.com/kyma-project/kyma/components/remote-environment-broker/pkg/apis/applicationconnector/v1alpha1"
+
+	clientset "github.com/kyma-project/kyma/components/remote-environment-broker/pkg/client/clientset/versioned"
+	"github.com/kyma-project/kyma/tests/ui-api-layer-acceptance-tests"
+	"github.com/kyma-project/kyma/tests/ui-api-layer-acceptance-tests/client"
+	"github.com/kyma-project/kyma/tests/ui-api-layer-acceptance-tests/waiter"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/kyma-project/kyma/tests/ui-api-layer-acceptance-tests/dex"
 	"github.com/kyma-project/kyma/tests/ui-api-layer-acceptance-tests/graphql"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-type reMutationData struct {
-	Name        string            `json:"name"`
-	Description string            `json:"description"`
-	Labels      map[string]string `json:"labels"`
+const (
+	reReadyTimeout = time.Second * 45
+)
+
+type remoteEnvironment struct {
+	Name                  string
+	Description           string
+	Labels                map[string]string
+	Services              []remoteEnvironmentService
+	EnabledInEnvironments []string
+	Status                string
+}
+
+type remoteEnvironmentService struct {
+	Id                  string
+	DisplayName         string
+	LongDescription     string
+	ProviderDisplayName string
+	Tags                string
+	Entries             remoteEnvironmentEntry
+}
+
+type remoteEnvironmentEntry struct {
+	Type        string
+	GatewayUrl  string
+	AccessLabel string
 }
 
 type reDeleteMutation struct {
 	Name string `json:"name"`
 }
 
 type reCreateMutationResponse struct {
-	ReCreateMutation reMutationData `json:"createRemoteEnvironment"`
+	ReCreateMutation remoteEnvironment `json:"createRemoteEnvironment"`
 }
 
 type reUpdateMutationResponse struct {
-	ReUpdateMutation reMutationData `json:"updateRemoteEnvironment"`
+	ReUpdateMutation remoteEnvironment `json:"updateRemoteEnvironment"`
```

Comment:
Wrong type - please use `reMutationOutput` type, which should be equal to previous `reMutationData`